core/tauri-runtime-wry/Cargo.toml

```diff
@@ -13,7 +13,7 @@ exclude = [ "CHANGELOG.md", "/target" ]
 readme = "README.md"
 
 [dependencies]
-wry = { version = "0.24.1", default-features = false, features = [ "file-drop", "protocol" ] }
+wry = { git = "https://github.com/tauri-apps/wry", branch = "v0.24.1", default-features = false, features = [ "file-drop", "protocol" ] }
```

Comment:
TODO: switch to published version when WRY is published and before this PR is merged